src/02 - Object types.md

```diff
@@ -333,6 +333,8 @@ The following notation is used in the tables below:
 |      | ref             | refSekundaerKlasse            |          |             |           |  |      |      |   x   |      |       x       | <p>&bull; references a **Klasse** from any **Klassifikasjonssystem**</p>                                                                                                                                                                                                                               |                                                                                                                                                        |
 |      | ref             | refEksternId                  |          |             |           |  |      |      |   x   |      |       x       | <p>&bull; *unlink* not supported in Documaster</p>                                                                                                                                                                                                                                                     | &nbsp;                                                                                                                                                 |
 |      | ref             | refNoekkelord                 |          |             |           |  |      |      |   x   |      |       x       | <p>&bull; *unlink* not supported in Documaster</p>                                                                                                                                                                                                                                                     | &nbsp;                                                                                                                                                 |
+|      | ref             | refAvskrivning                |          |             |           |  |      |      |   x   |      |       x       |                                                                                                                                                                                                                                                                                                        | &nbsp;                                                                                                                                                 |
```

Comment:
Unlinking `refAvskrivning`/`refTilknyttetAvskrivning` is not supported in Documaster.

Reply:
I partially agree. `refAvskrivning` cannot be unlinked, but `refTilknyttetAvskrivning` can. I've modified it accordingly.